### DISCUSSION TITLE
Bluetooth Jkbms reconnect every night at 1:06

### DISCUSSION THREAD

**Discussion by Honusnap** [Louisvdw/dbus-serialbattery#958] [Q&A]
Hi,
I suppose this is intended, but i did not find any reason why it would do that, every day at 1:06 i can hear my Jkmbs beeping cause the cerbo reconnect to it.

**Comment by mr-manuel** [comment]
You already opened an issue for that: https://github.com/Louisvdw/dbus-serialbattery/issues/957

**Reply by Honusnap** [reply]
Issue : 957 : It says the bug report is "invalid" ... i can't see why it's invalid.

Everything i've noticed is in there ... JKBMS reconnect (bip) every night at 1:06 and i can also see the lack of datas in my HomeAssistant at this same hour.

**Reply by mr-manuel** [reply]
It would help you, if you read the instructions :-)

**Reply by Honusnap** [reply]
I've read the instruction, installing once more the nightly too see if a bug has been corrected is not really a method and certainly not a reason to tag a bug report as "invalid". This bug report is on the stable version.
